theano/gof/cmodule.py

```diff
@@ -2000,6 +2025,20 @@ def join_options(init_part):
                     _logger.info("g++ -march=native equivalent flags: %s",
                                  GCC_compiler.march_flags)
 
+            # Find working march flag:
+            default_detected_flag = GCC_compiler.march_flags[0][7:]
+            march_flags_to_try = [default_detected_flag, 'corei7-avx', 'corei7', 'core2']
+            for march_flag in march_flags_to_try:
+                test_flags = GCC_compiler.march_flags[:]
+                test_flags[0] = '-march=' + march_flag
+                result = try_march_flag(test_flags)
+                if result[0] and result[1]:
+                    GCC_compiler.march_flags[0] = '-march=' + march_flag
+                    GCC_compiler.march_flags[-1] = '-mtune=' + march_flag
+                    break
+            if not result[0] or not result[1]:
+                raise ValueError('Could not find a working march')
```

Comment:
Don't raise an error, do: GCC_compiler.march_flags = []

We need it to still work on computer that aren't X86.